Initial draft charter.

dvorak42 opened the pull request:
Rendered preview: https://raw.githack.com/antifraudcg/antifraudcg.github.io/66fb7a4ff9be32c8ef9c0d102e70e2bd8c04f979/charter.html

npdoty commented:
I didn't bring it up during the call as it seemed obvious, but since it's in this draft: "being mindful of user security, privacy and accessibility" seems like a step back from the initial suggestion of "improving" those qualities. Just mindfulness suggests that the goal of the group is just to be aware when choosing to undermine user privacy or security or accessibility. I hope the CG already has a shared goal in actively supporting those properties, not just taking notice of them, and that we can detail that shared goal from the beginning in the group charter.

"supporting" might be an alternative that would work, if "improving" seems too aspirational or may not apply in every case.

npdoty commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+    </p>
+    <p>
+      The group welcomes participation from anti-fraud service providers,
+      common targets of unwanted traffic, browser vendors, web privacy
```
I'm not sure "web privacy advocates" is a common category (although I like the idea!), it might just be "privacy advocates" or "privacy and consumer advocates".

dvorak42 replied:
👍

npdoty commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+      The Community Group will discuss issues that server operators (network
+      and web server admins) and anti-fraud service providers face in this
+      area, as well as ideas for new web features and APIs intended to be
+      implemented in browsers or similar user agents.
```
I'm not sure we want to limit the scope to implementation in browsers and user agents. Many web features include implementation on the server side as well, or could involve servers/issuers/relying parties etc. 

The out-of-scope section directly notes that features that don't involve the client at all are out-of-scope (+1).

philippp replied:
If members identify means to advance our charter's goals via a design pattern (or protocol) that leverages existing browser components, should this proposal not be in scope? Constraining the solution set to ideas that require browser changes seems limiting in this regard. Broadening this to "as well as ideas for new web features, protocols, and APIs intended to be implemented in browsers or similar user agents" would accommodate these proposals.

npdoty commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+      the changes to the web platform and impact of the proposal.
+    </p>
+    <p>
+      Work items are proposals that can lead to interoperable standards/APIs.
```
Early sections of the charter suggest that some deliverables of the group will be documentation of scenarios and use cases, which wouldn't be direct feature/API proposals.

dvorak42 replied:
I wonder if we should have work items encompass both types of deliverables, and then the last stage (standardization for standards/APIs) doesn't apply or some other path is described for non-API "work items".

npdoty commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+      item, a proposal should be sent out to the CG mailing list, and there
+      must be at least two supporters of the proposal. For work items involving
+      web-exposed APIs, at least one supporter must be a browser vendor (as an
+      indication of interest in implementation).
```
As raised on the call, I don't think this explicit browser vendor requirement is necessary, given that there is a more general requirement in the next paragraph for sufficient likelihood of implementation.

npdoty commented:
Voting is included in several parts of the charter or suggested work mode for the group: initially to choose chairs, to address any decisions where there are sustained objections, and to make any changes to the charter. Repeated use of voting as a substantive decision process is counter to the consensus work mode; different stakeholder groups are unlikely to have similar numbers of organizations represented (for example, civil society groups were an extremely small number at the initial meeting); and voting is not even especially definable/workable in the Community Group context (as anyone could generate any number of organizations to join just to influence a vote).

If voting is genuinely intended to be a last resort, then it shouldn't be the first thing we do or something used in several places in the charter. We could instead choose chairs by consensus (with the help of the Community Group Development Lead if initial consensus is especially hard to evaluate), change or add new chairs by consensus, modify the charter by consensus, and allow chairs to record group decisions even when there is a sustained objection (as in W3C Process).

ianbjacobs commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+      Goals
+    </h2>
+    <p>
+      The mission of the Anti Fraud Community Group is to identify and define
```
```suggestion
      The mission of the <a href="https://www.w3.org/community/antifraud/">Anti Fraud Community Group</a> is to identify and define
```

ianbjacobs commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+    </h2>
+    <p>
+      Meetings will be held biweekly on {TODO: INSERT DATE}, lasting for one
+      hour. Meetings will be run by one of the Chairs and will request scribes
```
```suggestion
      hour. Meetings will be run by one of the Chairs, who will request scribes
```

ianbjacobs commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+    <p>
+      Meetings will be held biweekly on {TODO: INSERT DATE}, lasting for one
+      hour. Meetings will be run by one of the Chairs and will request scribes
+      to take meeting notes, which will be posted in the
```
```suggestion
      to take meeting notes that will be made publicly available in the
```

ianbjacobs commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+      <a href="#decision">consensus</a> that the work item is ready to
+      be migrated to a standards body, the Chairs will collaborate with the
+      Community Group to prepare a Community Group Report on the item, and with
+      the Editors of the item to forward it and the Report to the appropriate
```
It is not clear to me how "it" (the work item) is distinguished from the CG Report. Would it suffice to say that the group prepares a Report on the topic and then disseminates it to the relevant parties?

ianbjacobs commented on charter.html:
```diff
@@ -0,0 +1,330 @@
+      should be proposed for adoption as a work item. To be adopted as a work
+      item, a proposal should be sent out to the CG mailing list, and there
+      must be at least two supporters of the proposal. For work items involving
+      web-exposed APIs, at least one supporter must be a browser vendor (as an
```
Earlier in the charter is this text: "Features which don’t interact with the browser (or similar user agent) should be proposed in other standards bodies (IETF, etc)." That sentence suggests that that all proposals must be something like "web-exposed APIs." If so, is it necessary to say "For work items involving web-exposed APIs"?

dvorak42 replied:
Is intended to not include things like documentation deliverables that might not end up as web-standardizable. Tried updating the wording.

michaelficarra commented on charter.html:
```diff
@@ -0,0 +1,333 @@
+      Proposals
+    </h3>
+    <p>
+      Anyone can submit proposals to the proposals repository. If there is
```
This should link to the proposals repository.

dvorak42 replied:
👍

michaelficarra commented on charter.html:
```diff
@@ -0,0 +1,333 @@
+      <b>Participants announce their candidacies</b>: Participants have 14 days
+      to announce their candidacies. No two candidates can be from the same
+      organization. If there are three or fewer candidates, they become the
+      Chairs. If there are more than three candidates, a vote is held. (If
```
The voting system should be specified here. The restriction of one vote per organisation should be moved to this paragraph, or this sentence should be moved to the next paragraph.

michaelficarra commented on charter.html:
```diff
@@ -0,0 +1,333 @@
+      consulted for guidance).
+    </p>
+    <p>
+      <b>Participants vote</b>: The individuals who receive the most votes, no
```
"most votes" here is implying a certain class of voting system is being used. Maybe change it to something more generic like "the three winners of the election are appointed Chairs".

dvorak42 replied:
Updated with one potential approach.

michaelficarra commented on charter.html:
```diff
@@ -0,0 +1,333 @@
+      to announce their candidacies. No two candidates can be from the same
+      organization. If there are three or fewer candidates, they become the
+      Chairs. If there are more than three candidates, a vote is held. (If
+      there are no candidates, the Community Development Lead should be
```
Can we link the term "Community Development Lead" to more information on this role and how to contact them?

michaelficarra commented on charter.html:
```diff
@@ -0,0 +1,333 @@
+    </p>
+    <p>
+      If there is sustained opposition without changes to the issue, and the
+      proposer still wishes to proceed, the Chairs will hold a vote in the
```
```suggestion
      proposer still wishes to proceed, the Chairs will hold an approval vote in the
```

michaelficarra commented on charter.html:
```diff
@@ -0,0 +1,341 @@
+      consulted for guidance).
+    </p>
+    <p>
+      <b>Participants vote</b>: Each organization will have 3 votes that can
```
I would at least want to allow each organisation to vote for as many or as few candidates as they like. See [multiwinner approval voting](https://en.wikipedia.org/wiki/Multiwinner_approval_voting).

Alternatively, [single transferable vote](https://en.wikipedia.org/wiki/Single_transferable_vote) is better though more work to run.

ianbjacobs commented on charter.html:
```diff
@@ -0,0 +1,342 @@
+      Proposals
+    </h3>
+    <p>
+      Anyone can submit proposals to the
```
Should this be "Any participant"?

dvorak42 replied:
👍

ianbjacobs commented on charter.html:
```diff
@@ -0,0 +1,342 @@
+      must be at least two supporters of the proposal. For work items intended
+      to become a web-exposed API, at least one supporter should be a browser
+      vendor (as an indication of interest in implementation).
+    </p>
```
At this point a state diagram might be helpful, or a comprehensive labeling of states (e.g., in a definition list). The charter refers to a number of states:
* New proposal
* Withdrawn proposal
* Closed proposal
* Proposed work item
* Adopted work item

I assume there would also be something like a "closed work item"; the charter does not speak to that (and I think it's fine that it not do so until we have more experience).

In short, it would be helpful to describe the states and processes for entering them.

michaelficarra commented on charter.html:
```diff
@@ -119,7 +119,7 @@ <h3 id="proposals">
     </h3>
     <p>
-      Anyone can submit proposals to the
+      Any participant submit proposals to the
```
```suggestion
      Any participant may submit proposals to the
```

michaelficarra commented on charter.html:
```diff
@@ -0,0 +1,342 @@
+    <style>
+      body {
+        max-width: 60em;
+        margin: auto;;
```
```suggestion
        margin: auto;
```

michaelficarra commented on charter.html:
```diff
@@ -152,6 +152,18 @@ <h3 id="proposals">
+    <p>
+      <ul>
+        <li>New Proposal - A document put forward by a group of <b>author(s)</b> to be considered by the CG.</li>
+        <li>Closed Proposal - A document that has been withdrawn from consideration of the CG (by the <b>author(s)</b> or CG consensus.</li>
```
```suggestion
        <li>Closed Proposal - A document that has been withdrawn from consideration of the CG (by the <b>author(s)</b> or CG consensus).</li>
```

missing close paren

ianbjacobs reviewed (commented):
The following proposal integrates the lifecycle sentences with the bulleted list. The goal was to simplify, remove some potential inconsistencies (e.g., is "in scope" determined by chair or CG consensus?), and shrink the text. This is proposed as a replacement for the current "Proposals" section.

<h3 id="lifecycle">Proposal and Work Item Lifecycle</a>

* New Proposal - A document put forward by a group of author(s) to be considered by the CG. Any participant may submit proposals to the proposals repository. If there is substantial documentation/work, the proposal authors can ask the Chairs to set up a dedicated repository for a proposal. Proposals should generally include an Explainer document describing the changes to the web platform and impact of the proposal.

* Closed Proposal - A document that has been withdrawn from consideration of the CG (1) by the author(s), or (2) if the number of authors has dropped to zero, or (3) by CG consensus that the proposal is out of scope. The Chairs will check-in with active proposals to ensure the set of active authors is kept up-to-date.

Proposals that have community group support and result in a deliverable &mdash;either as interoperable standards/APIs or documentation&mdash; can become CG work items.

* Proposed Work Item - A document proposed to become a CG work item. To be adopted as a work item, a proposal should be sent out to the CG mailing list, and there must be at least two supporters of the proposal. For work items intended to become a web-exposed API, at least one supporter should be a browser vendor (as an indication of interest in implementation). Chairs should work with the authors to help ensure that a proposed work item is within the scope of the CG and that there is sufficient likelihood that the work item will be implemented in the ecosystem.

* Work Item - When consensus has been established, proposed work items become official work items of the CG for further iteration, driven by a group of assigned editor(s) who are generally the proposal authors.  

* Completed Work Item - A document that the CG is satisfied with and may be sent on to a working group for standardization.

npdoty commented:
Sure, we could look at the comments I raised:
https://github.com/antifraudcg/antifraudcg.github.io/pull/3#issuecomment-1000445900 (supporting security, privacy and accessibility)
https://github.com/antifraudcg/antifraudcg.github.io/pull/3#issuecomment-1000470476 (consensus rather than voting)

dvorak42 commented:
Switched to supporting text and updated the amendments to charter process to use consensus first. Left chair elections as a voting process, since without chairs to figure out what consensus is, establishing consensus is too handwavy

dvorak42 commented:
Going to merge the current state for the meeting, folks should continue to file issues or PRs for further changes.
